subscription_manager/subscription.py

```diff
@@ -38,6 +38,10 @@ def __str__(self):
             f"{self.comment})"
         )
 
+    @property
+    def next_payment_date(self):
+        return self.get_next_payment_date()
```

Comment:
please add cache in case function was called

Reply:
Will be done in another pull request in Subscription class tests.
Here I check that client is able to receive next payment date from the list of subscriptions